### DISCUSSION TITLE
Error 413 when uploading pictures >8MB

### DISCUSSION THREAD

**Discussion by PaulR282** [netbox-community/netbox#14168] [Help Wanted!] [answered]
### NetBox version

v3.6.4

### Python version

3.10

### Steps to Reproduce

1. Increase `FILE_UPLOAD_MAX_MEMORY_SIZE` (I have also increased `DATA_UPLOAD_MAX_MEMORY_SIZE`) to more than 10MB
2. Attach an Image that is bigger than 8MB to a rack

### Expected Behavior

The image should be uploaded

### Observed Behavior

`The connection was reset`
The log says 413

**Comment by squintfox** [comment]
@PaulR282 Are you using netbox-docker by chance?

**Comment by PaulR282** [comment]
> @PaulR282 Are you using netbox-docker by chance?

Yes, I am.

**Comment by squintfox** [comment, marked as answer]
We went through the same thing.  It's not a NetBox issue, it's a web server setting in the docker config.  You need to add this setting to the docker/nginx-unit.json file.

```
"settings": {
  "http": {
    "max_body_size": <int in bytes>
  }
}
```

Then you need to rebuild the container and have the file copied over.  We did something like this in the Dockerfile:

`COPY docker/nginx-unit.json /etc/unit/`

That's been working for us nicely.

**Comment by PaulR282** [comment]
> We went through the same thing. It's not a NetBox issue, it's a web server setting in the docker config. You need to add this setting to the docker/nginx-unit.json file.
> 
> ```
> "settings": {
>   "http": {
>     "max_body_size": <int in bytes>
>   }
> }
> ```
> 
> Then you need to rebuild the container and have the file copied over. We did something like this in the Dockerfile:
> 
> `COPY docker/nginx-unit.json /etc/unit/`
> 
> That's been working for us nicely.

Ok, Thanks. I will try it out tomorrow
